WooCommerce/WooCommerceTests/Authentication/NoWooErrorViewModelTests.swift

```diff
@@ -111,18 +111,13 @@ final class NoWooErrorViewModelTests: XCTestCase {
                                             showsConnectedStores: false,
                                             stores: stores,
                                             onSetupCompletion: { _ in })
-        let rootViewController = UIViewController()
         let noWooController = ULErrorViewController(viewModel: viewModel)
-        let navigationController = UINavigationController()
-        navigationController.viewControllers = [rootViewController, noWooController]
 
         // When
         viewModel.didTapSecondaryButton(in: noWooController)
 
         // Then
         XCTAssertFalse(stores.isAuthenticated)
-        XCTAssertEqual(navigationController.viewControllers.count, 1)
-        XCTAssertFalse(navigationController.topViewController is ULErrorViewController)
```

Comment:
> One unit test case relies on popToRootViewController which no longer works in iOS 18 + Xcode 18 unit tests. I found someone else reported the issue [forums.developer.apple.com/forums/thread/759640](https://forums.developer.apple.com/forums/thread/759640) and Apple asked to report it. For now, I removed this part of the test.

Sounds good. Testing the popping of the nav stack was handy but also a bit "smelly". It would be good if `NoWooErrorViewModel` was agnostic of its presentation mode so that it's up to whichever component presents it to handle how to dismiss the screen the view model drives. But I think a lot of this will have to be reworked when, inevitably, this will be updated to Swift UI.